src/docs/development/packages-and-plugins/using-packages.md

```diff
@@ -207,47 +214,45 @@ additional dependency options are avaialble:
 
 * **Git** dependency on a package in a folder: By default Pub assumes the
   package is located in the root of the Git repository. If that is not the
-  case, you can specify the location with the `path` argument. For example:
-  ```
+  case, specify the location with the `path` argument. For example:
+
+  ```yaml
   dependencies:
     package1:
       git:
         url: git://github.com/flutter/packages.git
         path: packages/package1
   ```
 
-  Finally, you can use the `ref` argument to pin the dependency to a
+  Finally, use the `ref` argument to pin the dependency to a
   specific git commit, branch, or tag. For more details, see
-  [Pub Dependencies]({{site.dart-site}}/tools/pub/dependencies).
+  [Package Dependencies]({{site.dart-site}}/tools/pub/dependencies).
 
 ## Examples
 
+If you are new to using packages, the following examples walk
+you through the necessary steps.
+
 ### Example: Using the CSS Colors package {#css-example}
 
 The [`css_colors`]({{site.pub-pkg}}/css_colors) package
-defines color constants for the CSS colors, allowing you to use them
+defines color constants for CSS colors, so they can be used
 wherever the Flutter framework expects the `Color` type.
 
 To use this package:
 
 1. Create a new project called 'cssdemo'
```

Comment:
Add a period, and change the single quotes to back ticks.
I think a lot of these steps don't have periods, but should.